curvefs/src/client/fuse_client.cpp

```diff
@@ -858,8 +858,6 @@ CURVEFS_ERROR FuseClient::FuseOpReadDirPlus(fuse_req_t req, fuse_ino_t ino,
         return ret;
     }
 
-    ::curve::common::UniqueLock lgGuard = inodeWrapper->GetUniqueLock();
```

Comment:
Test the case of reading a directory and inserting a file in the directory at the same time

Reply:
1. shell one: touch file in a dir
#!/bin/bash
i=1
while [ $i -le 100 ]; do
    touch /home/nbs/elk/data2/dir1/file$i
    let i++
done

echo "done"

2. shell two: ls -l the dir when create files in it
#!/bin/bash

while :; do
    ls -l  /home/nbs/elk/data2/dir1 >> ls.result
done

The result looks as expected:
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file1
total 0
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file1
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file2
total 0
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file1
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file2
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file3
total 0
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file1
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file2
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file3
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file4
total 0
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file1
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file2
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file3
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file4
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file5
total 0
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file1
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file2
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file3
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file4
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file5
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file6
total 0
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file1
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file2
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file3
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file4
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file5
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file6
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file7
total 0
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file1
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file2
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file3
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file4
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file5
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file6
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file7
-rw-r--r-- 1 nbs netease 0 Oct 26 20:24 file8
total 0
....